starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,53 @@
 // Names and Input
+var hacker1 =  'Pablo';
+console.log(`The Driver's name is ${hacker1}`);
 
+var hacker2 = window.prompt('Enter navigator\'s name');
+console.log(`Navigator's name is ${hacker2}`);
 
-//Conditionals
 
+//Conditionals
+if(hacker1.length > hacker2.length){
+    console.log(`The Driver has the longest name, it has ${hacker1.length} characters`);
+}else if(hacker2 > hacker1){
```

Comment:
Just a reminder, but don't forget to put .length at the end of hacker2 and hacker1